interface/patient_file/front_payment.php

```diff
@@ -401,6 +401,11 @@ function calcTaxes($row, $amount)
     WHERE encounter = ?", array($payrow['encounter']));
     $frow = $facilityService->getById($tmprow['facility_id']);
 
+ if($_REQUEST['radio_type_of_payment']=='pre_payment'){
+  // if pre_payment is being made then use the main office address added by Sherwin 7/21/2017
+   $frow = sqlQuery("SELECT * FROM `facility` WHERE name != '' LIMIT 1");
+   }
```

Comment:
if above is the case, then would do the following to code above:
``````
    if ($_REQUEST['radio_type_of_payment'] != 'pre_payment')
        $tmprow = sqlQuery("
        SELECT facility_id
        FROM form_encounter
        WHERE encounter = ?", array($payrow['encounter']));
    } else {
        // if pre_payment is being made then use the main office address - added by Sherwin 7/21/2017
        $tmprow = sqlQuery("SELECT * FROM `facility` WHERE name != '' LIMIT 1");
    }
    $frow = $facilityService->getById($tmprow['facility_id']);